cPP_DSC.adoc

```diff
@@ -1023,6 +1023,94 @@ FCS_CKM.2.1:: The TSF shall distribute cryptographic keys in accordance with a s
++ 
+*_Cryptographic key archival, backup, and escrow* - TOEs often perform cryptographic key archival to manage limited memory resources inside its own security boundaries.  Often the TOE encrypts the cryptographic keys prior to saving them to storage that is close by on the same device as the TOE and that can be accessed quickly. TOEs may perform cryptographic key backups into storage that is meant for longer term keeping.  Backup storage is often not on the same device as the TOE and may be physically hundreds of miles away.  In practice, the TOE encrypts the keys using an approved method before sending them to backup storage.  TOEs may perform cryptographic key escrow in which it entrusts a third party with access to the private or secret keys.  In practice, the TOE protects the keys using a cryptographic key access method agreed upon with the escrow agent before sending them._
++
+*_Cryptographic key recovery* - This refers to the retrieval of cryptographic keys from either archival, backup, or escrow locations.  In each case, the TOE uses the agreed upon cryptographic key access method agreed upon._
++
+*_Modifications to attributes of cryptographic keys* - Often during the creation of cryptographic keys, the TOE assigns certain attributes to them.  This may include allowed key access methods, authentication methods, and other items as discussed above in Creation of cryptographic keys.  The TOE may require authentication for authorization to modify these attributes._
++
+*_Cryptographic key destruction* - The attributes of the cryptographic key should indicate the necessary conditions needed to destroy it.  Often this will be an authentication indicating authorization to destroy it.  This SFR would apply to keys stored in the TOE, and FCS_CKM.6 would apply for destruction.  Otherwise, the destruction of keys stored outside the TOE is not enforceable by this SFR._
++
+*_Key Access Methods_*
++
+_This SFR lists several methods for protecting the confidentiality of keys prior to authorizing their use, archival, backup, escrow, or recovery.  This protection is often afforded only to the private or secret portion of cryptographic keys.  Some methods offer integrity protection as well.  These methods may not apply to modification of attributes especially if the attributes do not need confidentiality.  These methods may not apply to destruction since a TOE can only destroy keys within its boundaries, and it is premused these keys are already in the boundary._
```

Comment:
On key destruction outside the TOE boundary: Should we link that to some objective of the environment or OSP?